codespell_lib/_codespell.py

```diff
@@ -413,8 +413,17 @@ def parse_options(args):
         for key in config['codespell']:
             # Add option as arg.
             cfg_args.append("--%s" % key)
+            # get lines of value
+            val_lines = config['codespell'][key].splitlines()
+            # strip comments from lines
+            filtered_lines = []
+            for line in val_lines:
+                line = line.split('#', 1)[0].strip()
+                if line != "":
+                    filtered_lines.append(line)
```

Comment:
This manual solution concerns me a bit.  Stdlib functions always account for more possible errors and other things going wrong.  For example, given the following:
```ini
[codespell]
skip = "##d.txt##"
```
`val` below will be `"`, the result of manually stripping `"##d.txt##"` of the first `#` and everything after it.

Reply:
I decide to remove support for inline comments as `configparser` in Python stops handling these comments by default, and other libraries (like `coveragepy`) do not process these comments either.